src/click/__init__.py

```diff
@@ -84,6 +86,7 @@ def __getattr__(name: str) -> object:
             DeprecationWarning,
             stacklevel=2,
         )
+        cov.taken(1)
         return _BaseCommand
```

Comment:
At every if loop, there is an implicit else statement, if you would not go into an if loop, then you would end up at another branch for which you would check the next if loop and so on.
```suggestion
        cov.taken(1)
        return _BaseCommand
    else:
        cov.taken(2)
```